src/scripts/sidebar.js

```diff
@@ -192,12 +192,20 @@ class SideBar extends EventEmitter {
         document.body.classList.add('hide-server-list');
         localStorage.setItem('sidebar-closed', 'true');
         this.emit('hide');
+        if (process.platform === 'darwin') {
+            [].forEach.call(document.getElementsByTagName('webview'),
```

Comment:
If you use `querySelectorAll('webview')` you can use `forEach` like `document.querySelectorAll('webview').forEach(...` instead of using `[].forEach`